src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,117 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_START = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_START";
+    public static final String ACTION_STOP  = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP";
+    public static final String ACTION_STOP2 = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP2";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        switch (intent.getAction()) {
+            case ACTION_START:
+                SyncthingHelper.startSynchthingService(context);
+                break;
+
+            case ACTION_STOP:
+                SyncthingHelper.stopSynchtingService(context);
+                break;
+
+            case ACTION_STOP2:
+                SyncthingHelper.stopSynchtingService2(context);
+                break;
```

Comment:
Typos fixed ...

Well, when I created this class I had the idea that it might me nice to have the function isolated from the use so it can be reused more easily in later cases.
If you ever need to stop Syncthing for other reasons there then is a well tested way to do so. And then the isolation would be handy and would make refactoring (moving the functions to other places) more easy.
Otherwise it isn't reallly needed in the current use-case yet.

I'm not shure, but maybe these functions could be moved to the Application-object then? That's the place I would look for such a function in the first place.